tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "es6",
     "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
```

Comment:
Was this needed to get things typechecking for some reason? Generally I don't think this flag's use is encouraged if you're not writing Angular code that relies on it.

Reply:
Nope, that's leftover from trying something. Will remove.